CMake/SENSEIConfig.cmake.in

```diff
@@ -7,12 +7,18 @@ set(SENSEI_VERSION_DEVEL "@SENSEI_VERSION_DEVEL@")
 include(CMakeFindDependencyMacro)
 
 if (NOT SENSEI_DIR)
-  if (@SENSEI_BUILD@)
-    set(SENSEI_DIR "@CMAKE_BINARY_DIR@")
-  else()
-    set(SENSEI_DIR "@CMAKE_INSTALL_PREFIX@")
-  endif()
-endif()
+  # Use relocatable lookup for SENSEI_DIR
+  if (@SENSEI_INSTALL_RELOCATABLE@)
+    set(SENSEI_DIR "${CMAKE_CURRENT_LIST_DIR}/../../")
```

Comment:
@kwryankrattiger Revisiting this. I may have been wrong about this not working generally. If you can confirm that it works for both builds and installs, then your original was fine. I apologize about the mistake on my part.

Reply:
There are cases where it would break if someone was installing sensei, and then moving the cmake configuration files to a different location than the prescribed installation directory, and leaving the rest of the installation where it was installed. If this is not something that we need to support, then I can remove the old code.

The case with the `@CMAKE_BINARY_DIR@` works with both methods since we explicitly put the configuration file two directories up in the build tree.